### DISCUSSION TITLE
Crate claim?

### DISCUSSION THREAD

**Discussion by solmidnight** [rust-lang/crates.io#8937] [General]
I previously claimed https://crates.io/crates/editor under @brynnbrancamp. I have since changed my name as I have grown as a person. No less, I still have longer term plans for this crate. I would like to claim https://crates.io/crates/handshake

It has no repository, Author does not maintain crates. And all versions are yanked.

My proposal is implementing the Handshake domain protocol. I would like to use .api tld's I bought on namecheap for my backend. I will maintain this library and do what is needed and what I see fit to implement the protocol true to Rust.

Thank you,
Sol Midnight

**Comment by Turbo87** [comment]
> I previously claimed https://crates.io/crates/editor under @brynnbrancamp. I have since changed my name as I have grown as a person. No less, I still have longer term plans for this crate.

crates.io no longer allows long-term name squatting (see https://crates.io/policies), which is why the crate was deleted at some point. I see that you reregistered it, but again without any content. be aware that we may delete it again in the near future if we don't see any activity.

> I would like to claim https://crates.io/crates/handshake

Since https://github.com/rust-lang/rfcs/pull/3646 the crates.io team no longer mediates crate transfers. If you would like to use the name you will have to contact the author of that package and have them transfer the name to you.

**Reply by solmidnight** [reply]
Okay, thank you for letting me know. I will make sure to build horizontally and not squat, and I will choose names that are not taken. Apologies if I have caused any troubles.
